apple/user_access.md

```diff
@@ -0,0 +1,19 @@
+# Apple Account User Access
+
+All mozilla apple (mac/iOS) developers will need an apple account. We should try
+as much as possible only give out permissions to their @m.c accounts. Personal
+accounts should be avoided in case the developer leaves the company and we don't
+delete the apple account.
+
+## Permissions
+Roles are confusing!
```

Comment:
These are a bit vague, and it may be more helpful to the general Relenger if you guide them. E.g., "For developers, check these boxes." If you don't want to document every little thing, then it may come down to "if someone asks for something out of these guidelines, then ask ____". If you don't want to be pinged about these questions, then you likely want to document the edge cases.